recipes/diligentgraphics-volk/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+class VolkConan(ConanFile):
+    name = "diligentgraphics-volk"
```

Comment:
Where does diligentgraphics come from?

```suggestion
    name = "zeux-volk"
```